packages/base-styles/_z-index.scss

```diff
@@ -10,13 +10,18 @@ $z-layers: (
 	".block-editor-block-list__layout .reusable-block-indicator": 1,
 	".block-editor-block-list__block-selection-button": 22,
 	".components-form-toggle__input": 1,
-	".components-panel__header.edit-post-sidebar__panel-tabs": -1,
-	".edit-post-sidebar .components-panel": -2,
```

Comment:
These were not referenced anywhere. It appears they were leftovers from a previous iteration of sticky inspector tabs (maybe the version mentioned here: https://github.com/WordPress/gutenberg/pull/28003#pullrequestreview-562500165)